site/assets/_scss/site/common/_fonts.scss

```diff
@@ -8,10 +8,11 @@
 }
 
 @font-face {
-  font-family: "Metropolis";
+  font-family: "Metropolis-Light";
   src: local("Metropolis Light"), local("Metropolis-Light"),
     url("#{$path-fonts}/Metropolis/Metropolis-Light.woff") format("woff");
-  font-weight: $font-weight-light;
+  font-weight: normal;
```

Comment:
@amymanion pretty sure you can remove font weight and font style from the @font-face definition since those should go on the css rules themselves

Reply:
Removed